### ISSUE TITLE
Feedback Nikolas 👨‍💻

### ISSUE THREAD

**Issue by KarelZe** [KarelZe/thesis#399]
## Detailed comments

 * Page 2 (Trade Classification in Option Markets):
   > Together with the trade size rule, their second rule, which classifies trades with a trade size matching the size of the bid or ask quote, can substantially improve the performance of classical algorithms

   Der Satz ist irgendwie bisschen verschachtelt. Was genau kann die performance verbessern? Die second rule? Ist das die trade size rule? Oder geht es um beide rules? Dann würde hier sowas wie "together with..., which..., these can substantially..." oderso, damit klar ist, dass beide ein improvement versprechen.

 * Page 3 (Trade Classification Using Machine Learning): "minimising" -- Das ist britisches Englisch, bist du da konsistent? Amerikanisches Englisch wäre "minimizing".

 * Page 3 (Trade Classification Using Machine Learning): "neighbour" -- Auch britisches English, amerikanisch wäre "neighbor"). Ich würde damit davon ausgehen dass du konsistent britisches Englisch nutzt und ab jetzt sachen markieren die in amerikanischem Englisch geschrieben sind.

 * Page 4 (Trade Classification Using Machine Learning): "paper" -- ich würde allgemein "work" statt "paper" in einer Master Thesis nutzen.

 * Page 4 (Trade Classification Using Machine Learning): "Trade Reporting and Compliance Engine (TRACE)" -- Scheint anklickbar zu sein aber führt nirgendwo hin. Ich gehe davon aus du fügst noch ein Abkürzungsverzeichnis hinzu am Ende?

 * Page 4 (Trade Classification Using Machine Learning): "," -- Das Komma kann weg

 * Page 6 (Basic Rules):
   > This section presents basic classification rules, that may be used for trade classification independently or integrated into a hybrid algorithm.

   Hier ist wohl ein Zeilenumbruch reingerutscht

 * Page 8 (Tick Test): "and calls" -- Das klingt falsch. Was genau calls for more sophisticated approaches? Grauer et al.? Der fakt, dass beide approaches nicht mit quote-based approaches mithalten können? Glaube da brauchst du entweder "call" oder sowas wie "this calls" oderso.

 * Page 16 (Selection of Approaches): "guided computational constraints" -- "guided by computational constraints", oder?

 * Page 19 (Selection of Approaches): "the form FT-Transformer" -- "in the form OF..."

 * Page 19 (Selection of Approaches): "The findings Ronen" -- "The findings OF..."

 * Page 29 (Token Embedding): "," -- das komma brauchst du hier nicht

 * Page 30 (Token Embedding): "," -- komma brauchst du hier nicht

 * Page 45 (Gradient Boosted Trees With Self-Training): "," -- Das komma kann weg
 * Page 51 (Data Collection): "To arrive at labelled sample" -- To arrive at labelled samples oder to arrive at a labelled sample (entweder mehrzahl oder "a" hinzufügen)

 * Page 52 (Data Collection):
   > Our definition is consistent with the in Grauer et al. (2022, p. 8).

   with THAT in grauer et al., oder with the ONE FROM gauer et al. (oder ähnliches)

 * Page 54 (Data Preprocessing): "estimators have" -- Ich nehme an, hier fehlt ein Komma, um den Nebensatz abzuschließen.

 * Page 55 (Data Preprocessing): "there-off" -- thereof

 * Page 62 (Training of Supervised Models):
   > 13Code is available at https://github.com/KarelZe/thesis Experiments are tracked at https: //wandb.ai/fbv/thesis.

   Die Links funktionieren beide nicht

 * Page 70 (Training of Supervised Models): "resulting in a resulting" -- einmal resulting in zu viel

 * Page 74 (Hyperparameter Tuning):
   > The loss surface of the GBRT trained on the feature set including option features is the least fragmented. While the validation accuracy of the best combinations improves significantly to 76.99 %, worst trials even underperform these of smaller feature sets. Based on this observation we conjecture, that more data does not per se improve the model and that models require a thoughtful tuning procedure. By this means, our conclusion contradicts the one of Ronen et al. (2022, p. 14), who find no advantage in tuning tree-based ensembles for trade classification. Results are tabulated in Table 3.

   Der Absatz kam doch so 1zu1 vorher schonmal vor (auf Seite 66)? Ist das Absicht?

 * Page 75 (Hyperparameter Tuning): "16FS 1: https://wandb.ai/fbv/thesis/runs/37lymmzc FS 2: https://wandb.ai/fbv/thesis/runs/324v3uv5 FS 3: https://wandb.ai/fbv/thesis/runs/t55nd8r0" -- Auch hier wieder, die Links funktionieren nicht (könnte aber sein dass man sich hier zunächst in W&B einloggen muss? Kenn die Seite nicht. Kannst mir gerne was darüber auf Signal erzählen :))
